CHANGES.rst

```diff
@@ -28,6 +28,8 @@ Bug fixes:
 - Remove superfluos dependency on zope.formlib
   [jensens]
 
+- removed unittest2 dependency
+  [kakshay21]
```

Comment:
This is right, but wrongly placed. See these change log entries are for the already released version 1.2.3. You want to add it under the ``1.2.4 (unrelease)`` on the ``bugfixes`` section. And there,as it will be the first entry, remove the placeholder one